exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Packet.java

```diff
@@ -183,6 +183,24 @@ public InetAddress getDst_ip() {
     return getIPAddress(false);
   }
 
+  public String getSrc_ip_String() {
```

Comment:
Also please rename methods here (including methods above) according to JCC, for example to something like `getSourceHostAdress`.

Reply:
Fixed